jetty-xml/src/main/resources/org/eclipse/jetty/xml/configure_10_0.dtd

```diff
@@ -286,7 +286,7 @@ System Property Element.
 This element allows JVM System properties to be retrieved as
 part of the value of elements such as Set, Put, Arg, etc.
 The name attribute specifies the property name and the optional
-default argument provides a default value.
+default attribute provides a default value.
```

Comment:
leave this as argument, as default can be ...

``` xml
<SystemProperty name="Test">
  <Default>value</Default>
</SystemProperty>
```

Reply:
"The name attribute ..." so if `name` is an attribute (but can be an element), then also `default` should use the same naming. 
Either they are both "attribute" (and I think it's ok because the first line below enumerate the possible child elements), or we use "attribute/element" everywhere.